### ISSUE TITLE
Crash when joining server

### ISSUE THREAD

**Issue by ichttt** [lumien231/Random-Things#283]
The game crashes as soon as you reconnect. Using version 4.1.2 (latest for 1.12.2). Crashreport:
https://pastebin.com/jDSvhmPp

**Comment by lumien231** [comment]
Do you have a redstone interface connected to a notification interface? Could you maybe upload your world? Also try to remove all mods besides rt.

**Comment by ichttt** [comment]
Yes, I tried reproducing it, and this time I got kicked as soon as I tried connecting the redstone interface to the notification interface. I upload a newly create world [here](http://www.mediafire.com/file/vsd0pn82rekv28b/world.zip) with rt as the only mod. It does not happen every time when connecting the two interfaces, and (when only using rt) never when connecting with rt only. Seems to be a race condition, as login in my modpack takes as long as 15-20 seconds. I couldn't find a way to reproduce it every time though.
